test/integration/features/user_creates_from_url.spec.js

```diff
@@ -49,7 +41,7 @@ describe('authenticated e2e-user', function() {
 
   afterAll(function() {
     projectHelpers.deleteAllProjects();
-    resetExtensions();
+    browser.executeScript("window.OPENSHIFT_CONSTANTS.CREATE_FROM_URL_WHITELIST = ['openshift'];");
```

Comment:
Do we want symmetrical `setupEnv` and `tearDownEnv` for this?  No strong pref, otherwise everything looks good to me.  Yay removes a bunch of code!

Reply:
-135 lines, +3.  thats pretty good tradeoff 😄

Reply:
Unfortunately it doesn't work since we reload the page when we navigate :(

Reply:
Mmm. I have a bunch of the nav updated to use in-page navigation in the e2e test overhaul PR...